packages/zero/package.json

```diff
@@ -114,12 +114,16 @@
       "default": "./out/zero/src/server.js"
     },
     "./server/adapters/postgresjs": {
-      "types": "./out/zero-server/src/adapters/postgres.d.ts",
+      "types": "./out/zero-server/src/adapters/postgresjs.d.ts",
       "default": "./out/zero/src/adapters/postgresjs.js"
     },
-    "./server/adapters/drizzle-pg": {
```

Comment:
This is technically a breaking change, but I don't know how many people are using this entrypoint.